nucleus/annotation.py

```diff
@@ -893,6 +899,48 @@ def to_payload(self) -> dict:
         return payload
 
 
+@dataclass
+class SceneCategoryAnnotation(CategoryAnnotation):
+    """A scene category annotation.
+
+    ::
+
+        from nucleus import CategoryAnnotation
+
+        category = SceneCategoryAnnotation(
+            label="running",
+            reference_id="scene_1",
+            taxonomy_name="action",
+        )
+
+    Parameters:
+        label (str): The label for this annotation.
+        reference_id (str): User-defined ID of the scene to which to apply this annotation.
+        taxonomy_name (Optional[str]): The name of the taxonomy this annotation conforms to.
+          See :meth:`Dataset.add_taxonomy`.
+    """
+
+    @classmethod
+    def from_json(cls, payload: dict):
+        return cls(
+            label=payload[LABEL_KEY],
+            reference_id=payload[REFERENCE_ID_KEY],
+            taxonomy_name=payload.get(TAXONOMY_NAME_KEY, None),
+        )
+
+    def to_payload(self) -> dict:
+        payload = {
+            LABEL_KEY: self.label,
+            TYPE_KEY: SCENE_CATEGORY_TYPE,
+            GEOMETRY_KEY: {},
+            REFERENCE_ID_KEY: self.reference_id,
+            METADATA_KEY: {},
```

Comment:
Is there a specific reason why we don't support `metadata` right now?

Reply:
No reason, just omitted it for the first iteration of the implementation